app/src/editor/canvas/index.jsx

```diff
@@ -172,15 +172,52 @@ const CanvasEditComponent = class CanvasEdit extends Component {
     }
 
     loadNewDefinition = async (hash) => {
-        const { replace } = this.props
+        const { replace, undo } = this.props
         const module = CanvasState.getModule(this.props.canvas, hash)
-        const newModule = await sharedServices.getModule(module)
+        try {
+            const newModule = await sharedServices.getModule(module)
 
-        if (!this.unmounted) {
-            replace(() => CanvasState.updateModule(this.props.canvas, hash, () => newModule))
+            if (!this.unmounted) {
+                replace(() => CanvasState.updateModule(this.props.canvas, hash, () => newModule))
+            }
+        } catch (error) {
+            console.error(error.message)
```

Comment:
This removes the stack trace from the reported error… `console.error(error)` should do the right thing?